src/app/(write)/writeTest/page.tsx

```diff
@@ -1,37 +1,103 @@
-import React from 'react';
+'use client';
+
+import React, { useRef, useState } from 'react';
+
+import { Descendant } from 'slate';
 
 import DayButton from '@/components/textEditor/DayButton';
-import SlateEditor from '@/components/textEditor/SlateEditor';
+import SlateEditor, { SlateEditorRef } from '@/components/textEditor/SlateEditor';
+
+function Page() {
+  const days = 5;
+  const [activeDay, setActiveDay] = useState<number>(1); // Default to the first day being active
+  const [editorValues, setEditorValues] = useState<Descendant[][]>(
+    Array(days).fill([{ type: 'paragraph', children: [{ text: '' }] }])
+  );
+  const editorRefs = useRef<(SlateEditorRef | null)[]>(Array(days).fill(null));
+  const [serializedValues, setSerializedValues] = useState<string[]>(Array(days).fill(''));
+
+  const handleDayButtonClick = (day: number) => {
+    setActiveDay(day);
+  };
+
+  const handleEditorChange = (index: number, value: Descendant[]) => {
+    const newValues = [...editorValues];
+    newValues[index] = value;
+    setEditorValues(newValues);
+  };
```

Comment:
```suggestion
  const handleEditorChange = (currentIndex: number, newValue: Descendant[]) => {
    const newValues = editorValues.map((editorValue, index) => index === currentIndex? newValue: editorValue);
    setEditorValues(newValues);
  };
```

이런 방법도 있을 것 같아 제시만 해봅니다~